scripts/modules/b2b-account/quotes.js

```diff
@@ -57,9 +91,89 @@ define([
     });
 
     var QuoteModel = Backbone.MozuModel.extend({
-      });
+    });
+
+    var QuotesGridCollectionModel = MozuGridCollection.extend({
+        mozuType: 'quotes',
+        defaultSort: 'submittedDate desc',
+        columns: [
+            {
+                index: 'name',
+                displayName: 'Quote Name',
+                sortable: false
+            },
+            {
+                index: 'expirationDate',
+                displayName: 'Expiration Date',
+                sortable: true,
+                displayTemplate: function (value) {
+                    var date = new Date(value);
+                    return date.toLocaleString();
+                }
+            },
+            {
+                index: 'submittedDate',
+                displayName: 'Created',
```

Comment:
Should `submittedDate` really map to `Created` exactly? Seems it would be better to call this column `Submitted`. Can you double check the requirements here? Thanks.

Reply:
changed label to submittedDate